doobie/src/test/scala/za/co/absa/fadb/doobie/DoobieOptionalResultFunctionTest.scala

```diff
@@ -30,7 +31,7 @@ class DoobieOptionalResultFunctionTest extends AnyFunSuite with DoobieTest {
 
   private val getActorById = new GetActorById()(Integration, new DoobieEngine(transactor))
 
-  test("Retrieve actor by id from database") {
+  test("Retrieve actor by id from database", IntegrationTestTag) {
```

Comment:
isn't it possible to tag the class, instead of individual test?

Reply:
This option is not supported for ScalaTest test framework.
I have applied QA type system base solution used in atum-service --> requires test file name refactoring.